redwood-treehouse-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -97,13 +133,18 @@ private class CellContentSource<A : AppService>(
   }
 }
 
-private class TableViewCell(private val view: UIView) :
-  UITableViewCell(UITableViewCellStyle.UITableViewCellStyleDefault, null) {
-  init {
-    addSubview(view)
-  }
+private class CollectionViewCell(frame: CValue<CGRect>) : UICollectionViewCell(frame) {
+  var view: UIView? = null
 
   override fun layoutSublayersOfLayer(layer: CALayer) {
-    view.setFrame(bounds)
+    view!!.setFrame(bounds)
+  }
+
+  // If this isn't included, an 'Initializer is not implemented' exception is thrown.
+  @Suppress("OVERRIDE_DEPRECATION")
+  override fun initWithFrame(frame: CValue<CGRect>): UICollectionViewCell {
+    return CollectionViewCell(frame)
   }
```

Comment:
Not blocking, but I'd love to remove this function. Without it, I'm getting the exception:

```
Terminating app due to uncaught exception 'Initializer is not implemented', reason: 'initWithFrame: is not implemented in ESK_kobjc4'
```

Reply:
I think this is fine to include